.openshift-ci/build/Dockerfile.cpaas

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/stackrox-io/apollo-ci:collector-0.3.44-1-gb00ffc52af as driver-cache
+
+COPY /kernel-modules/MODULE_VERSION /MODULE_VERSION
+COPY /.openshift-ci/build/build-driver-cache.sh /scripts/build-driver-cache.sh
+
+# Pull the CPaaS built drivers from GCP
+RUN export MODULE_VERSION="$(cat /MODULE_VERSION)" && \
+    mkdir -p "/kernel-modules/${MODULE_VERSION}" && \
+    gsutil -m rsync -r \
+        "gs://collector-modules-osci-public/cpaas//$MODULE_VERSION/" \
+        "/kernel-modules/$MODULE_VERSION/" || true
+
+RUN /scripts/build-driver-cache.sh
+
+FROM quay.io/stackrox-io/collector:3.11.0-slim
```

Comment:
Does this need to parameterised or are we expecting to update this tag for each release?

Reply:
For now, it's handled by OSCI and it runs with the collector version that is currently built by the repo. If we want to add a way to run this manually, we can parametrize it to use the output of `make tag` instead of a fixed tag, but I don't know if we want to spend the extra time at this point, WDYT?